src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
-body {
+$size: 100px;
+$gap: 1px;
+$day-of-week:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  max-width: $size * 7 + $gap * 10;
+  gap: 1px;
+  flex-wrap: wrap;
+  margin: 100px auto;
+
+  @for $show from 28 through 31 {
+    &--month-length--#{$show} {
+      :nth-child(-n + #{$show}) {
+        width: $size;
+        height: $size;
+        background-color: #eee;
+        border: 1px solid black;
+      }
+    }
+  }
+
+  @for $day-none from 28 through 31 {
+    &--month-length--#{$day-none} > :not(:nth-child(-n+#{$day-none})) {
+      display: none;
+    }
+  }
+
+  @each $day, $number in $day-of-week {
+    &--start-day--#{$day} > :first-child {
+      margin-left: ($size + $gap) * $number;
+    }
+  }
+
+  &__day {
+    font-size: 30px;
+    text-align: center;
+    line-height: 100px;
+    cursor: pointer;
+    transition-duration: 0.5s;
```

Comment:
When you use transition specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/376da4c5-a41a-43c8-b533-bd7d78b53cd6)